GameData/ROCapsules/PartConfigs/Apollo D2/D2ServiceModule.cfg

```diff
@@ -0,0 +1,139 @@
+PART
+{
+	name = ROC-D2ServiceModule
+	module = Part
+	author = Mcdouble, Vader111
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROCapsules/Assets/McDouble/D2/D2ServiceModule
+	}
+
+	scale = 1
+	rescaleFactor = 1.0
+
+	node_stack_top = 0.0, 1.375, 0.0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
+
+	CoMOffset = 0.0, -1.4, 0.0
+
+	attachRules = 1,0,1,1,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = FuelTank
+	subcategory = 0
+
+	title = D-2 Service Module
+	manufacturer = General Electric
+	description = Service Module for D-2 vehicle, carries fuel and other vital systems.
+	tags = D2, apollo, GE
+
+	bulkheadProfiles = size2
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 1.2
+	dragModelType = default
+	maximum_drag = 0.20
+	minimum_drag = 0.15
+	angularDrag = 2
+	crashTolerance = 10
+	breakingForce = 250
+	breakingTorque = 250
+	maxTemp = 673.15
+	skinMaxTemp = 773.15
+	fuelCrossFeed = true
+	emissiveConstant = 0.6 // not too absorptive for reentry
+	thermalMassModifier = 1.0
+	skinMassPerArea = 4.0
+	radiatorHeadroom = 0.5
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 10000.0
```

Comment:
This should be 12000; in testing a few years ago I found 10,000 was insufficient for the craft's stated mission (lunar orbital).